src/common/reactor.cc

```diff
@@ -254,7 +254,7 @@ namespace Pistache::Aio
             // Note: poller_reg_unreg_mutex is already locked (by
             // Listener::run()) before calling here, so it is safe to call
             // handlers_.forEachHandler here
-                
```

Comment:
Why are non-test files changed?

Reply:
This is clang formatting - the lines are empty lines containing only whitespace, and the whitespace has been removed. No code impact obviously.